examples/fuzzing/stm32f407/fuzz_stm32f407.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# 
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+
+import sys, os
+sys.path.append("../../..")
+
+from binascii import hexlify
+
+from qiling.core import Qiling
+from qiling.const import QL_VERBOSE
+from qiling.extensions.mcu.stm32f4 import stm32f407
+from qiling.extensions.afl import ql_afl_fuzz_custom
+from unicorn import UC_ERR_OK, UcError
+from qiling.arch.arm import QlArchARM
+
+
+def watch(ql:Qiling):
+    r0 = hex(ql.arch.regs.read('r0'))
+    if r0 != '0x0':
+        print(f"R0 == {r0}")
+    # hex(ql.arch.regs.read('r0'))
+    print(ql.mem.read(0x8003320, 0xd))
+
+def found_passwd(ql:Qiling):
+    # print('right')
+    os.abort()
+
+def wrong_test(ql:Qiling):
+    # print('no')
+    ql.emu_stop()
+
+def fuzz_cb(ql:Qiling):
+    if isinstance(ql.arch, QlArchARM):
```

Comment:
Could use `ql.ostype` to determine this, or just `hasattr(ql.arch, 'effective_pc')` if you want to avoid the extra `import`.